.docker/entrypoint/spec/helpers/spec_jekyll_commander.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module PlantUml
+    module Specs
+      module Helpers
+        class SpecJekyllCommander
```

Comment:
I'd like to see a comment or two on what this class does, unless the functionality makes sense for an experienced Ruby developer.